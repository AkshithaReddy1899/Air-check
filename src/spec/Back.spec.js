import React from 'react';
import matches from './test-utils';
import Back from '../components/Pages/Back';
import { MemoryRouter } from 'react-router-dom';

describe('Details page', () => {
  it('DeailsPage matched snapshot', () => {
    matches(<MemoryRouter><Back /></MemoryRouter>)
  });
});
