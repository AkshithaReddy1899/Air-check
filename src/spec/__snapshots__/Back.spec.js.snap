// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Details page DeailsPage matched snapshot 1`] = `
<a
  href="/"
  onClick={[Function]}
>
  Back
</a>
`;
